app-config.yaml

```diff
@@ -16,6 +16,10 @@ backend:
     credentials: true
   csp:
     connect-src: ["'self'", 'http:', 'https:']
+  reading:
+    allow:
+      - host: example.com
+      - host: '*.mozilla.org'
```

Comment:
I think we should have `github.com` in the default configuration, as we have some API entities that are using this feature: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/apis/spotify-api.yaml

Reply:
As long as the default readers are added as well (`GithubUrlReader` for example), they automatically add a config entry for the public site whether you actually have it in your config or not. So if I understand things correctly it'll still work, just not fall down to the `FetchUrlReader`.

Reply:
Ah, that make sense